dali/test/python/reader/test_numpy.py

```diff
@@ -150,7 +150,7 @@ def NumpyReaderPipeline(
         np.float_,
         np.complex64,
         np.complex128,
-        np.complex_,
+        complex,
```

Comment:
Oh wow, `np.complex_ == np.complex128`, `np.complex == complex`. I can see why they removed the np.float and np.complex.:)

So the change is not exactly synonymous, but if the tests pass, I guess it's fine.

Reply:
It passes as we don't support complex anyway so whatever that is, as long as `all_numpy_types` and `unsupported_numpy_types` use the same types we are good.